### ISSUE TITLE
Support nodejs10.x runtime in Lambda

### ISSUE THREAD

**Issue by hhakkala** [terraform-providers/terraform-provider-aws#8627]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Specifying `nodejs10.x` as function runtime currently results in validation error:

```
$ terraform -v
Terraform v0.11.13
+ provider.aws v2.10.0

$ terraform apply
Error: module.default.aws_lambda_function.default: expected runtime to be one of [dotnetcore1.0 dotnetcore2.0 dotnetcore2.1 go1.x java8 nodejs4.3 nodejs4.3-edge nodejs6.10 nodejs8.10 provided python2.7 python3.6 python3.7 ruby2.5], got nodejs10.x
```

### New or Affected Resource(s)

* aws_lambda_function

### Potential Terraform Configuration

```hcl
resource "aws_lambda_function" "default" {
  function_name = "MyFunction"
  ...
  runtime = "nodejs10.x"
}
```

### References

* #4019

**Comment by acburdine** [comment]
I've got a PR open to add support here: #8622

**Comment by bflad** [comment]
Support for the new `nodejs10.x` runtime in the `aws_lambda_function` resource has been merged and will be released with version 2.11.0 of the Terraform AWS Provider later this week. 👍

**Comment by bflad** [comment]
This has been released in [version 2.11.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2110-may-17-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by BarakBD-Globality** [comment]
The docs should be updated to reflect support for the new nodejs version (nodejs8 is being deprecated).
https://www.terraform.io/docs/providers/aws/r/lambda_function.html#basic-example
I clicked `edit this page` but could not understand where to make the change.
